{"name": "BTC 创建托管账户name为空", "status": "broken", "statusDetails": {"message": "Exception: 请求异常", "trace": "self = <test_create_fail.Test_create_custodial_account object at 0x10ee57cd0>, test_title = 'BTC 创建托管账户name为空', name = ''\nnetworkCode = 'BTC', status_check = 400, code_check = 2100000\n\n    @allure.story(\"Create Custodial Account Fail!\")\n    @allure.title('{test_title}')\n    @pytest.mark.parametrize('test_title,name,networkCode,status_check,code_check', test_data)\n    def test_create_account_custodial(self, test_title, name, networkCode, status_check, code_check):\n    \n        with allure.step(\"创建托管账户\"):\n>           custodial_account = Http.HttpUtils.create_custodial_account(name,networkCode)\n\nTestcase/Test_Account/Test_Custodial_Account/test_create_fail.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nname = '', networkCode = 'BTC'\nAuthorization = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIxNTg1MTAyNzkxNDY2NDg3ODEwIiwiZXhwIjoxNjk4Mjg4OTM1LCJpYXQiOjE2NjY3NTI5MzUsInVzZXJJZCI6MTU4NTEwMjc5MTQ2NjQ4NzgxMH0.4o43r1upI2Qzcs46wf3gqb2Kq6kq0I7qGST6qPspfjY'\n\n    @staticmethod\n    # 创建托管账户\n    def create_custodial_account(name: str, networkCode: str, Authorization=token):\n        url = url_ + '/vault/accounts/custodial'\n        headers = {\n            \"Content-Type\": \"application/json\",\n            \"Authorization\": Authorization\n        }\n        body = {\n            \"name\":name,\n            \"networkCode\": networkCode\n        }\n    \n        logger.info('\\n'+\"<-----Create custodial Account------>\"+\"\\n\"+\"Url:\"+url+'\\n\\n'+'Headers:'+json.dumps(headers)+'\\n\\n'+'Body:'+json.dumps(body))\n        res = requests.post(url=url, json=body, headers=headers, timeout=timeout_)\n        if res.status_code == 200:\n            logger.info('\\n'+\"<-----Create custodial Response----->\"+\"\\n\"+(res.text))\n            return res\n        else:\n            logger.error('\\n'+'<-----Create custodial Response Error----->'+(res.text))\n>           raise Exception(\"请求异常\")\nE           Exception: 请求异常\n\nCommon/Http.py:187: Exception"}, "steps": [{"name": "创建托管账户", "status": "broken", "statusDetails": {"message": "Exception: 请求异常\n", "trace": "  File \"/Users/lilong/Documents/Test_Api/Testcase/Test_Account/Test_Custodial_Account/test_create_fail.py\", line 41, in test_create_account_custodial\n    custodial_account = Http.HttpUtils.create_custodial_account(name,networkCode)\n  File \"/Users/lilong/Documents/Test_Api/Common/Http.py\", line 187, in create_custodial_account\n    raise Exception(\"请求异常\")\n"}, "start": 1668063236641, "stop": 1668063237025}], "attachments": [{"name": "log", "source": "8eeda55a-0f06-4c00-bd12-5971b290e9ac-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_title", "value": "'BTC 创建托管账户name为空'"}, {"name": "name", "value": "''"}, {"name": "networkCode", "value": "'BTC'"}, {"name": "status_check", "value": "400"}, {"name": "code_check", "value": "2100000"}], "start": 1668063236641, "stop": 1668063237028, "uuid": "a2f39946-bace-479d-a913-95d42c662bcd", "historyId": "9cf2e516eb137a674775ec70dd39a3f1", "testCaseId": "ad4896695e8d2db5ae6499b340ef0484", "fullName": "Testcase.Test_Account.Test_Custodial_Account.test_create_fail.Test_create_custodial_account#test_create_account_custodial", "labels": [{"name": "feature", "value": "Create Custodial Account!"}, {"name": "story", "value": "Create Custodial Account Fail!"}, {"name": "parentSuite", "value": "Testcase.Test_Account.Test_Custodial_Account"}, {"name": "suite", "value": "test_create_fail"}, {"name": "subSuite", "value": "Test_create_custodial_account"}, {"name": "host", "value": "lilongdeMacBook-Pro.local"}, {"name": "thread", "value": "18444-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase.Test_Account.Test_Custodial_Account.test_create_fail"}]}